pkg/graph/resolvers/system_intake.go

```diff
@@ -114,6 +115,26 @@ func SystemIntakeUpdate(ctx context.Context, store *storage.Store, fetchCedarSys
 	intake.HasUIChanges = null.BoolFromPtr(input.HasUIChanges)
 	intake.UsesAITech = null.BoolFromPtr(input.UsesAiTech)
 
+	acqMethods := pq.StringArray{}
+	for _, acqMethod := range input.SoftwareAcquisition.AcquisitionMethods {
+		acqMethods = append(acqMethods, acqMethod.String())
+	}
+	// TODO: NJD make this a function in model_helpers? ReverseConvertEnums?
```

Comment:
I found a little `Map` function using `samber/lo` that isn't much _less_ code than this, but avoids `append()`

```suggestion
	acqMethods := lo.Map(input.SoftwareAcquisition.AcquisitionMethods, func(acqMethod models.SystemIntakeSoftwareAcquisitionMethods, idx int) string {
		return acqMethod.String()
	})
```